packages/crayons-core/src/components/tab/tab.scss

```diff
@@ -3,24 +3,30 @@
   white-space: nowrap;
   cursor: pointer;
   align-items: center;
-  padding: 10px 8px;
+  padding: var(--fw-tab-padding-tb, 10px) var(--fw-tab-padding-lr, 8px);
```

Comment:
can u add these css variables as comment so that they show up in docs

Reply:
Done.